sched/semaphore/sem_waitirq.c

```diff
@@ -97,10 +97,6 @@ void nxsem_wait_irq(FAR struct tcb_s *wtcb, int errcode)
 
       sem->semcount++;
 
-      /* Indicate that the semaphore wait is over. */
-
-      wtcb->waitsem = NULL;
```

Comment:
Don't we need to set waitobj to NULL here?

Reply:
waitobj will set to NULL in nxsched_remove_blocked() function which will be called later.